common/src/Microsoft.Azure.IIoT.Agent.Framework/src/Agent/Default/Worker.cs

```diff
@@ -176,7 +186,7 @@ public class Worker : IWorker, IDisposable {
                 var workerHeartbeat = await GetWorkerHeartbeatAsync(_cts.Token).ConfigureAwait(false);
 
                 await _jobManagerConnector.SendHeartbeatAsync(
-                    new HeartbeatModel { Worker = workerHeartbeat }, _cts.Token).ConfigureAwait(false);
+                    new HeartbeatModel { Worker = workerHeartbeat }, GetDiagnosticInfo(), _cts.Token).ConfigureAwait(false);
```

Comment:
Let's try to keep length of the line under `120` characters.

Reply:
fixed